test/new-e2e/tests/windows/msi.go

```diff
@@ -0,0 +1,55 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+// Package windows contains helpers for Windows E2E tests
+package windows
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
This could be a `windowshelper` package, at the same level of `agent` and `install-test`
```
windows \
  - agent
  - installtest
  - windowshelpers [ or helpers ]
```
  
I would put the test at windows root actually, and have it using the helpers in `agent` and `helpers`

Reply:
this is the layout we decided to start with after speaking with WKIT. We'll see how it goes after we add more utilities and tests.